Dockerfile

```diff
@@ -15,6 +15,9 @@ COPY . /usr/src/app
 # Install app dependencies
 RUN npm install
 
+# Build project 
+RUN npm run build
```

Comment:
npm install should already do a build: https://github.com/PathwayCommons/app-ui/blob/ecbc2db7505a565570745bca69b8496714b2b760/package.json#L14